### PR TITLE
Fix lighting intensity

### PR DESCRIPTION
**Pull request by PeterJCLaw**
I'm guessing that this changed somehow when we rebuilt the arena this year.

The main change here is that the in-match intensity is decreased from 3.5 to 2.5. The other changes are small refactors ahead of pulling that value to a top level constant.

Fixes https://github.com/srobo/competition-simulator/issues/364.

*Before*
before

*After*

https://user-images.githubusercontent.com/336212/219198413-4501567e-19a4-4751-997f-cc38cdeb51e0.mp4

### CODE REVIEW

**Comment by WillB97** [comment]
Have you disabled "Ambient Occlusion" before making this change? When AO is enabled at any level it seems to add a significant amount of bloom which could have been your original issue.

**Comment by PeterJCLaw** [comment]
> Have you disabled "Ambient Occlusion" before making this change? When AO is enabled at any level it seems to add a significant amount of bloom which could have been your original issue.

Yes, I was working through the ["Running competition matches"](https://github.com/srobo/competition-simulator/#running-competition-matches) docs when I found this. Would be good to confirm that it's not just my machine though.

**Review by WillB97 — COMMENTED** [review]
This leave the pre-match dim quite bright, reduce that intensity to about 0.1

**Comment by PeterJCLaw** [comment]
https://user-images.githubusercontent.com/336212/216830693-bc10d1d7-d06f-4bcb-9d1b-931d277c0ce7.mp4

**Comment by WillB97** [comment]
It could do with being darker at the start.

File: controllers/lighting_controller/lighting_controller.py [hunk 1]
```diff
@@ -71,14 +75,25 @@ class LuminosityFade:
...
+        lighting=[ArenaLighting('SUN', intensity=MATCH_LIGHTING_INTENSITY)],
+        name="Fade-up",
+    ),
     LightingEffect(
         -0.1,
-        lighting=[ArenaLighting('SUN', intensity=1, colour=(1, 0, 0))],
+        lighting=[
+            ArenaLighting(
+                'SUN',
+                intensity=NON_MATCH_LIGHTING_INTENSITY,
```
Comment (WillB97) [inline controllers/lighting_controller/lighting_controller.py:93]: You need a non-zero intensity at the end so that the red colour shows.

Comment (PeterJCLaw) [reply controllers/lighting_controller/lighting_controller.py:95]: https://github.com/srobo/competition-simulator/pull/365/commits/914c35b6ae343913d7976d713d10132e640720d7

**Comment by PeterJCLaw** [comment]
https://user-images.githubusercontent.com/336212/219164335-9e9ccd48-d2df-4ddb-8aa7-819ec26205c6.mp4

File: controllers/lighting_controller/lighting_controller.py (hunk 1 above)
Comment (PeterJCLaw) [reply controllers/lighting_controller/lighting_controller.py]: https://github.com/srobo/competition-simulator/pull/365/commits/8e0c48ea41ec93754aca23bc9a1bca570063adc4

**Comment by PeterJCLaw** [comment]
https://user-images.githubusercontent.com/336212/219166753-5d96fd30-9d1a-466d-9fea-3e843f037fc2.mp4

**Comment by PeterJCLaw** [comment]
https://user-images.githubusercontent.com/336212/219198413-4501567e-19a4-4751-997f-cc38cdeb51e0.mp4

File: controllers/lighting_controller/lighting_controller.py (hunk 1 above)
Comment (PeterJCLaw) [reply controllers/lighting_controller/lighting_controller.py]: I think this is resolved now? #379 seemed to help a lot.

**Pull request merged**